.github/workflows/dart.yml

```diff
@@ -35,7 +33,7 @@ jobs:
             sdk: dev
 
     steps:
-        - uses: cedx/setup-dart@v2
+        - uses: dart-lang/setup-dart@v1
```

Comment:
its still using `release-channel` to config the channel, its different for this action